internal/actions/setvar_test.go

```diff
@@ -96,6 +96,16 @@ func TestSetvarEvaluate(t *testing.T) {
 			init:                     "TX.newvar=-%{tx.missingvar}",
 			expectInvalidSyntaxError: true,
 		},
+		{
+			name:                     "Non Numerical Operation - If the value starts with -",
+			init:                     "TX.newvar=----expected_value",
+			expectInvalidSyntaxError: false,
+		},
```

Comment:
should we include `expectNewVarValue` in the test swell?

Reply:
Done